test/testdata/sandbox_config_seccomp.json

```diff
@@ -47,7 +47,6 @@
 	},
 	"annotations": {
 		"owner": "hmeng"
-		%VALUE%
```

Comment:
@rhatdan, yeah, as this json isn't really meant to be "runnable" by ocic. Not sure what you mean by "You can put it back" though :confused: